source/_components/smartthings.switch.markdown

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
The SmartThings documentation should follow the new approach for the documentation for integrations. We decided to keep the things in the documentation together thus we don't want platform documentation for integration which are powered by a component. The platform details should be part of the component documentation. There are already examples present and this work much better than providing multiple times the same standard phrases.

Reply:
OK, I'll close this PR and update the other in-flight items.  A new PR will be needed to pull in lights.